Cirumflex_Combined.ino

```diff
@@ -1,12 +1,19 @@
 //Fixed colors set as globals. 03-Pattern fades between them.
 const int RANDOMnoisePIN = A5; // for random seed
 const int BRIGHTNESS = 1;      // Value pot pin number
-const int SLOWNESS = 3;        // Pot pin number; 0 is fast, 1023 is slow. 
+const int SLOWNESS = 3;        // Pot pin number; 0 is fast, 1023 is slow.
+
+// Which LED interface?
+#include "PWM_NativePin.h"
+PWM_NativePin_MEGA PWM;
+
 // Pins for LEDs in each zone
-const byte aLunitPins[3] = {13, 12, 11};    // Zone A    >> Zone 1   {R, G, B}
-const byte bLunitPins[3] = {2, 3, 4};       // Zone B    >> Zone 2   {R, G, B}
-const byte cLunitPins[3] = {8, 9, 10};      // Zone C    >> Zone 3   {R, G, B}
-const byte dLunitPins[3] = {7, 6, 5};       // Zone D    >> Zone 4   {R, G, B}
+// You have to go look in the PWM class to figure out the physical pin
+// On the mega, pwm1 is pin 2, and increments till pwm12
+const byte aLunitPins[3] = {PWM.pwm12, PWM.pwm11, PWM.pwm10};    // Zone A    >> Zone 1   {R, G, B}
+const byte bLunitPins[3] = {PWM.pwm1, PWM.pwm2, PWM.pwm3};       // Zone B    >> Zone 2   {R, G, B}
+const byte cLunitPins[3] = {PWM.pwm7, PWM.pwm8, PWM.pwm9};      // Zone C    >> Zone 3   {R, G, B}
+const byte dLunitPins[3] = {PWM.pwm6, PWM.pwm5, PWM.pwm4};       // Zone D    >> Zone 4   {R, G, B}
```

Comment:
Instead of saying "2", use an expression that will work if the physical pin number is different. e.g. on the TLC59711 the first pwm pin is 1.

Reply:
Dan,
Why do some RGB's have pin#'s ascending (pwm1, pwm2, pwm3) and some descending pwm6, pwm5, pwm4)?

Reply:
I think I get the point of your first comment. Again, should be a quick fix.
I think I just put the wires in first, and then figured out which pin number went with which color.

Reply:
Sounds like order is not important.